### PR TITLE
Refactor BaseCollection.download

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: resdk/resources/collection.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 import six
 
-from .base import BaseResource, DOWNLOAD_TYPES
```
Comment (dblenkus) [inline resdk/resources/collection.py]: Is ``DOWNLOAD_TYPES`` used anywhere else? Otherwise remove it from ``base``.

Comment (JureZmrzlikar) [inline resdk/resources/collection.py]: No, good point, I will remove it.

File: resdk/resources/collection.py [hunk 2]
```diff
@@ -88,32 +88,22 @@ def download(self, file_name=None, data_type=None, download_dir=None):
...
-        if data_type:
-            if isinstance(data_type, six.string_types) and data_type in DOWNLOAD_TYPES.keys():
-                # type is a shortcut (string)
-                data_type, output_field = DOWNLOAD_TYPES[data_type]
-            elif isinstance(data_type, tuple) and len(data_type) == 2:
-                data_type, output_field = data_type
-            else:
-                raise ValueError("Invalid argument value data_type.")
+        if file_type and not isinstance(file_type, six.string_types):
+            raise ValueError("Invalid argument value file_type.")
```
Comment (dblenkus) [inline resdk/resources/collection.py]: Put `file_type` in backticks.

**Comment by dblenkus** [comment]
Few minor comments, otherwise LGTM.

**Pull request merged**